airflow-core/src/airflow/jobs/triggerer_job_runner.py

```diff
@@ -95,7 +95,11 @@ def __init__(
     ):
         super().__init__(job)
         if capacity is None:
-            self.capacity = conf.getint("triggerer", "default_capacity", fallback=1000)
+            # PR 48032 renames the triggerer default_capacity to capacity. The precedence is capacity ->
+            # default_capacity -> 1000
+            self.capacity = conf.getint("triggerer", "capacity") or conf.getint(
+                "triggerer", "default_capacity", default=1000
+            )
```

Comment:
```suggestion
            self.capacity = conf.getint("triggerer", "capacity")
```

This is all you need. Your addition to deprecated_options will fall back to the old config option or the default, if not set explicitly.